valhalla/sif/dynamiccost.h

```diff
@@ -324,12 +324,17 @@ class DynamicCost {
                       (!forward && next_pred->edgeid() == cr->to_graphid()))) {
 
           if (current_time && cr->has_dt()) {
-            if (baldr::DateTime::is_restricted(cr->dt_type(), cr->begin_hrs(), cr->begin_mins(),
-                                               cr->end_hrs(), cr->end_mins(), cr->dow(),
-                                               cr->begin_week(), cr->begin_month(),
-                                               cr->begin_day_dow(), cr->end_week(), cr->end_month(),
-                                               cr->end_day_dow(), current_time,
-                                               baldr::DateTime::get_tz_db().from_index(tz_index))) {
+            LOGLN_WARN("Restricted: " + std::to_string(cr->begin_hrs()));
+            // TODO Possibly a bug here. Shouldn't both kTimedDenied and kTimedAllowed
```

Comment:
I haven't dug too closely but I would assume we should be handling both cases as is done in `IsRestricted` here.